examples/darwin-framework-tool/commands/common/CHIPCommandBridge.mm

```diff
@@ -76,15 +102,20 @@
         mControllers[identities[i]] = controller;
     }
 
-    // If no commissioner name passed in, default to alpha.
-    SetIdentity(mCommissionerName.HasValue() ? mCommissionerName.Value() : kIdentityAlpha);
-
-    ReturnLogErrorOnFailure(RunCommand());
-    ReturnLogErrorOnFailure(StartWaiting(GetWaitDuration()));
-
     return CHIP_NO_ERROR;
 }
 
+CHIP_ERROR CHIPCommandBridge::MaybeTearDownStack()
+{
+    CHIP_ERROR err;
+    if (IsInteractive()) {
+        return CHIP_NO_ERROR;
+    }
+    err = ShutdownCommissioner();
+    ReturnErrorOnFailure(chip::DeviceLayer::PlatformMgr().StopEventLoopTask());
```

Comment:
This doesn't make sense here, fwiw.  It'll happen automatically in commissioner shutdown.

I am pretty sure the `StartEventLoopTask` call in StartWaiting can also go away.

Reply:
Ok. Let me try it out with only controller startup and shutdown.

Reply:
Updated.